internal/alloycli/cmd_run.go

```diff
@@ -248,6 +248,8 @@ func (fr *alloyRun) Run(configPath string) error {
 		AdvertiseInterfaces: fr.clusterAdvInterfaces,
 		ClusterMaxJoinPeers: fr.ClusterMaxJoinPeers,
 		ClusterName:         fr.clusterName,
+		//TODO(add #issue number): graduate to GA once we have more confidence in this feature
```

Comment:
Should you do this before merge?

Reply:
Haha sorry, I mean should you open the issue and add the issue number here before merge